lib/vanagon/component.rb

```diff
@@ -290,6 +290,14 @@ def get_build_dir
       end
     end
 
+    def get_dependency_hash
+      obj = @options
+      [:version, :name].each do |var|
+        obj[var.to_s] = self.instance_variable_get("@#{var}")
```

Comment:
> I was only doing that just incase later on we want to get more data out.

Sure, but it's also not hard to add another line that plucks out the value from `self` and puts it into the map.

I don't like metaprogramming this simple thing; it has more negatives than positives.

I think we should simplify this.